src/EventStore.Common/Log/EventStoreLoggerConfiguration.cs

```diff
@@ -43,7 +44,9 @@ public class EventStoreLoggerConfiguration {
 			};
 		}
 
-		public static void Initialize(string logsDirectory, string componentName, string logConfig = "logconfig.json") {
+		public static void Initialize(string logsDirectory, string componentName, LogConsoleFormat logConsoleFormat,
+			int logFileSize, RollingInterval logFileInterval, int logFileRetentionCount, bool disableLogFile,
+			string logConfig = "logconfig.json") {
```

Comment:
the `logConfig.json` si a hidden confiugration option ?

Reply:
No it should be documented